packages/mergebot/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "dt-mergebot",
+  "version": "3.0.0",
+  "description": "The bot that manages DefinitelyTyped PRs.",
+  "author": "DT Maintainers",
+  "homepage": "https://github.com/DefinitelyTyped/dt-mergebot#readme",
+  "main": "dist/functions/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/DefinitelyTyped/dt-mergebot.git"
+  },
+  "bugs": {
+    "url": "https://github.com/DefinitelyTyped/dt-mergebot/issues"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=18"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.10.3",
+    "@azure/functions": "^4.4.0",
+    "@definitelytyped/old-header-parser": "npm:@definitelytyped/header-parser@0.0.178",
+    "@definitelytyped/utils": "^0.1.6",
+    "@octokit/webhooks-methods": "^4.1.0",
+    "dayjs": "^1.11.11",
+    "fast-json-patch": "^3.1.1",
+    "graphql": "15.8.0",
+    "prettyjson": "^1.2.5",
+    "yargs": "^17.7.2"
+  },
+  "devDependencies": {
+    "@octokit/graphql-schema": "^14.58.0",
+    "@octokit/webhooks-types": "^7.5.1",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^18.19.33",
+    "@types/prettyjson": "^0.0.33",
+    "@types/yargs": "^17.0.32",
+    "@typescript-eslint/eslint-plugin": "^7.9.0",
+    "@typescript-eslint/parser": "^7.9.0",
+    "apollo": "^2.34.0",
+    "azure-functions-core-tools": "^4.0.5700",
+    "esbuild": "^0.21.5",
+    "eslint": "^8.57.0",
+    "eslint-plugin-unicorn": "^52.0.0",
+    "jest": "^29.7.0",
+    "jest-environment-node": "^29.7.0",
+    "jest-file-snapshot": "^0.5.0",
+    "knip": "^5.15.1",
+    "rimraf": "^5.0.7",
+    "ts-jest": "^29.1.2",
+    "typescript": "^5.4.5"
+  },
+  "overrides": {
+    "apollo": {
+      "graphql": "15.8.0"
+    },
+    "apollo-language-server": {
+      "graphql": "15.8.0"
+    }
+  },
+  "scripts": {
+    "prestart": "npm run build",
+    "prepublishOnly": "npm run build",
+    "build": "tsc",
+    "bundle": "rimraf dist && esbuild --bundle --platform=node --format=cjs --target=node18 --external:@azure/functions-core --outfile=dist/functions/index.js src/functions/index.ts",
+    "watch": "tsc --watch",
+    "graphql-schema": "apollo client:codegen schema --target typescript --globalTypesFile=src/queries/schema/graphql-global-types.ts",
+    "create-fixture": "npm run build && node dist/commands/create-fixture.js",
+    "update-test-data": "npm run build && node dist/commands/update-test-data.js",
+    "update-all-fixtures": "npm run build && node dist/commands/update-all-fixtures.js",
+    "start": "func start",
+    "lint": "eslint --report-unused-disable-directives .",
+    "test": "jest"
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "/dist/"
+    ]
+  },
+  "packageManager": "npm@10.7.0+sha256.f443ed4364ea11ac5cf7cae7fb4731278c64dd6839093f8a46eabde0430e0fcd"
```

Comment:
You probably want to re-copy this; all of the stuff here is out of date because it uses npm

Reply:
done